COPY/etc/issue.template

```diff
@@ -1,4 +1,4 @@
-Welcome to the \P Virtual Appliance.
+Welcome to the Virtual Appliance.
```

Comment:
Yes, this is much nicer.
When this was added tried to get exact parity.

Good to go back and question if this is necessary.

I do still like having the ip address (`\i`) though.